packages/select/src/components/omnibar/omnibar.tsx

```diff
@@ -153,19 +145,9 @@ export class Omnibar<T> extends React.PureComponent<IOmnibarProps<T>, IOmnibarSt
         );
     };
 
-    private renderItems({ activeItem, filteredItems, handleItemSelect }: IQueryListRendererProps<T>) {
-        const { itemRenderer, noResults } = this.props;
-        if (filteredItems.length === 0) {
-            return noResults;
-        }
-        return filteredItems.map((item, index) =>
-            itemRenderer({
-                handleClick: e => handleItemSelect(item, e),
-                index,
-                isActive: item === activeItem,
-                item,
-            }),
-        );
+    private renderItems({ items, renderItem }: IQueryListRendererProps<T>) {
```

Comment:
not sure where `IQueryListRendererProps` is defined, but i think this should be `itemRenderer`